2-mandatory/2-currency-code-lookup.js

```diff
@@ -19,6 +19,15 @@ const COUNTRY_CURRENCY_CODES = [
 
 function createLookup(countryCurrencyCodes) {
```

Comment:
This function can be shortened a lot using .fromEntries() method.

function createLookup(countryCurrencyCodes) {
  let obj = Object.fromEntries(countryCurrencyCodes);
  return obj;
}